build.sh

```diff
@@ -296,6 +297,9 @@ while :; do
         verbose)
             __VerboseBuild=1
             ;;
+        staticliblink)
+            __CMakeExtraArgs="$__CMakeExtraArgs$staticCurl -DCMAKE_STATIC_LIB_LINK=1"
```

Comment:
`$staticCurl` is still present on this line; but it doesn't seem to be set by this file.  Since that will normally do nothing, but occasionally maybe break the world (if someone had that set in their own environment) we should snip that part out.